### PR TITLE
Fix user page in admin page

### LINKED ISSUE #11: Fix user page in admin page
### Describe the bug
At user page in admin page, it shows only the default fields; `username` and `password`. (There is another field; `team`.)

image

### Expected behavior
The `team` field should be shown correctly.

### To Reproduce
1. Run server using the command `python manage.py runserver`. (You should have already completed migrations.) 
2. Open a browser with the url `http://localhost:8000/admin/`.
3. Move the user page.

### Expected causes and correction methods
Referred to [Custom User Admin](https://docs.djangoproject.com/en/5.1/topics/auth/customizing/#custom-users-and-django-contrib-admin) and [ModelAdmin options](https://docs.djangoproject.com/en/5.1/ref/contrib/admin/#modeladmin-options), we need to define a custom user admin inherited from `UserAdmin`.

### Additional context
For ease of understanding, please refer to [this](https://tcitr-antoliny.tistory.com/35) blog.

### PR DESCRIPTION
**Pull request by BECATRUE**
This resolves #11. For details, please read the issue.

A user list page is as follows.

image

A user detail page is as shown below.

image

### CODE REVIEW

File: wlm_server/user/admin.py [hunk 1]
```diff
@@ -3,4 +3,31 @@
 
 from .models import User
 
-admin.site.register(User, UserAdmin)
+@admin.register(User)
+class CustomUserAdmin(UserAdmin):
```
Comment (kangz12345) [inline wlm_server/user/admin.py:7]: How about setting `last_login` field as read-only?

See also: [`readonly_fields`](https://docs.djangoproject.com/en/5.1/ref/contrib/admin/#django.contrib.admin.ModelAdmin.readonly_fields)

**Review by kangz12345 — CHANGES_REQUESTED** [review]

File: wlm_server/user/admin.py (hunk 1 above)
Comment (kangz12345) [reply wlm_server/user/admin.py:7]: See also: https://forum.djangoproject.com/t/how-to-set-last-login-to-read-only-in-the-admin-pages/29213

Comment (BECATRUE) [reply wlm_server/user/admin.py:7]: It looks much better👍

**Review by kangz12345 — APPROVED** [review]
LGTM

**Pull request merged**